packages/compass-crud/src/components/insert-json-document.jsx

```diff
@@ -63,15 +42,15 @@ class InsertJsonDocument extends Component {
 
     return (
       <div className={classnames(styles.editor)}>
-        <Ace
-          mode="json"
-          defaultValue={EDITOR_COMMENT}
-          value={value}
-          onChange={this.onChange.bind(this)}
-          theme="mongodb"
-          width="100%"
-          editorProps={{$blockScrolling: Infinity}}
-          setOptions={OPTIONS}
+        <Editor
+          variant={EditorVariant.EJSON}
+          placeholder={EDITOR_COMMENT}
```

Comment:
Do we want the `placeholder` to stay? Might be a nice feature?

```suggestion
          defaultValue={EDITOR_COMMENT}
```
or

```suggestion
          defaultValue={EDITOR_COMMENT}
          placeholder={EDITOR_COMMENT}
```

<img width="614" alt="Screen Shot 2022-04-27 at 4 28 25 PM" src="https://user-images.githubusercontent.com/1791149/165625353-00c8c9fe-2fae-43df-a13c-6e7b133abaec.png">
hmm getting some lag and no highlighting here. Restarting my instance, but may be a different mode or something, I'll remove this when it finishes building if it was a dep install issue.

Reply:
Ah no, I actually got mixed up a bit here. I thought `defaultValue` was the ace version of `placeholder`, because I wasn’t sure what `defaultValue` would mean otherwise (since we’re always passing `value` as well). I’m still not sure but it’s probably better if I don’t touch this for now. :slightly_smiling_face: 

The missing JSON highlighting (and the `foldStyle` issue below) was just a missing import, my bad :)

Reply:
If we are always passing value, we probably can remove the defaultValue prop, yeah. Although I'm wondering if there might be cases where having only a defaultValue passed can help with ace editor performace, maybe opting it out of the controlled component behavior can help with the lag we are noticing and in some cases we probably don't need it to be synced back and forth all the time